### PR TITLE
Translate document to zh_HANS/zh_HANT

### PR DESCRIPTION
**Pull request by Gemerz**
Hi, Samsung team, thank your guys to develop this awesome project make easily for our TV developer. So I translate to simple/traditional Chinese to let  more developers read the document and using project more handily.

### CODE REVIEW

File: doc/README_zh_HANS.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
+# WITs
+WITs 是一个非常有效的开发工具，很容易帮助开发属于你的 Tizen web 项目，并支持**2017+ Samsung tv**的机型。
+它可以非常快地将你开发期间的实时代码推送到电视机上。从而节省了每次你的打包及重新项目的当你运行在设备上。我们称为 `实时开发`。
```
Comment (prozanne) [inline doc/README_zh_HANS.md:3]: Hi :)
Thanks a lot for your nice effort!

Could you please brush up the sentence a bit?
Our Chinese member said it's better to be changed a bit.
从而节省了每次你的打包及重新项目的当你运行在设备上。

File: doc/README_zh_HANS.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+```
+
+#### 2.  安装WITs依赖
+
+```sh
+    $ cd ~/{path-to}/Wits
+    $ npm install -g
+```
+
+#### 3. 更改 `.witsconfig.json` 在 `Wits` 的目录内.
```
Comment (prozanne) [inline doc/README_zh_HANS.md:37]: #### 3. 更改 `.witsconfig.json` 在 `Wits` 的目录内.

Here as well.

File: doc/README_zh_HANS.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+The default `path` is `tizen-studio-data/profile/profiles.xml` on Mac and Windows both.
+配置对应的 Tizen Studio 证书 Profile  `path` profiles.xml所对应的路径到
```
Comment (prozanne) [inline doc/README_zh_HANS.md:40]: 40~42 as well.

File: doc/README_zh_HANS.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+![witss](https://user-images.githubusercontent.com/1733182/77503927-420b1000-6ea2-11ea-88f5-49ab0c5fc227.gif)
+
+### wits -w
+
+![witsw](https://user-images.githubusercontent.com/1733182/77503928-43d4d380-6ea2-11ea-8ece-4f5182cb7d6d.gif)
+
+## FAQ
+
+-   [WITs FAQ](https://github.com/Samsung/Wits/wiki/Frequently-Asked-Questions)
+-   [怎样使用Chrome inspector的调度模式](https://github.com/Samsung/Wits/wiki/Frequently-Asked-Questions#answer-1)
```
Comment (prozanne) [inline doc/README_zh_HANS.md:149]: 调度模式
Here also!

Thanks in advance!

File: doc/README_zh_HANS.md (hunk 1 above)
Comment (Gemerz) [reply doc/README_zh_HANS.md:3]: Hi @prozanne , thank for advices, that trans sentences which make some laggy,  I have translated like:
`每次您想查看项目如何在设备上运行时，都无需经历再次构建，打包和重新安装应用程序。`

File: doc/README_zh_HANS.md (hunk 2 above)
Comment (Gemerz) [reply doc/README_zh_HANS.md:37]: #### 3.  在 `Wits` 的目录下更改 `.witsconfig.json` , as well?

File: doc/README_zh_HANS.md (hunk 1 above)
Comment (prozanne) [reply doc/README_zh_HANS.md:3]: Cool!

File: doc/README_zh_HANS.md (hunk 3 above)
Comment (Gemerz) [reply doc/README_zh_HANS.md:40]: 40~42: are retranslated like this:

Tizen Studio的证书路径（profiles.xml）是在 **.witsconfig.json** 的 `path` 上配置,
而在mac与Windows系统上默认路径 `path` 都是 `tizen-studio-data/profile/profiles.xml` 。

Is that OK？

File: doc/README_zh_HANS.md (hunk 4 above)
Comment (Gemerz) [reply doc/README_zh_HANS.md:149]: sorry for input error 😭:
retranslated like this:
怎样使用Chrome inspector的开发模式。

File: doc/README_zh_HANS.md (hunk 3 above)
Comment (prozanne) [reply doc/README_zh_HANS.md:40]: cool!

File: doc/README_zh_HANS.md (hunk 2 above)
Comment (prozanne) [reply doc/README_zh_HANS.md:37]: Yes please!

**Comment by Gemerz** [comment]
hi @prozanne, Is all reviews are good, I just commit & push, thanks 😊.

File: doc/README_zh_HANS.md (hunk 4 above)
Comment (prozanne) [reply doc/README_zh_HANS.md]: It's ok! Thanks!!

**Review by prozanne — APPROVED** [review]
Looks good.
Thanks!

**Pull request merged**